src/main/java/org/springframework/cloud/deployer/spi/mesos/chronos/ChronosTaskLauncher.java

```diff
@@ -244,5 +249,13 @@ private Double deduceCpus(AppDeploymentRequest request) {
 		return override != null ? Double.valueOf(override) : properties.getCpu();
 	}
 
+	private Collection<Constraint> deduceConstraints(AppDeploymentRequest request) {
+		Set<Constraint> requestSpecific = StringUtils.commaDelimitedListToSet(request.getDeploymentProperties().get("spring.cloud.deployer.chronos.constraints"))
```

Comment:
This is a known inconsistency (see https://github.com/spring-cloud/spring-cloud-deployer-mesos/issues/41) both for Chronos and Marathon. I chose to be consistent with how it's currently done, but definitely weird. cc @trisberg